cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -545,16 +545,27 @@ public static KafkaBridge createKafkaBridgeCluster(String clusterCmNamespace, St
     /**
      * Generate empty Kafka MirrorMaker 2.0 ConfigMap
      */
-    public static KafkaMirrorMaker2 createEmptyKafkaMirrorMaker2Cluster(String clusterCmNamespace, String clusterCmName) {
-        return new KafkaMirrorMaker2Builder()
+    public static KafkaMirrorMaker2 createEmptyKafkaMirrorMaker2Cluster(String clusterCmNamespace, String clusterCmName, Integer replicas) {
+        KafkaMirrorMaker2Builder kafkaMirrorMaker2Builder = new KafkaMirrorMaker2Builder()
                 .withMetadata(new ObjectMetaBuilder()
                         .withName(clusterCmName)
                         .withNamespace(clusterCmNamespace)
                         .withLabels(TestUtils.map(Labels.KUBERNETES_DOMAIN + "part-of", "tests",
                                 "my-user-label", "cromulent"))
-                        .build())
-                .withNewSpec().endSpec()
-                .build();
+                        .build());
+
+        if (replicas != null) {
+            kafkaMirrorMaker2Builder
+                    .withNewSpec()
```

Comment:
I think this might cause issues if in the future someone adds something to the spec in the initial block and doesn't notice this overrides the whole spec. Maybe you could use here `editOrNewSpec` instead of `withNewSpec`